.gitignore

```diff
@@ -5,3 +5,4 @@ npm-debug.log
 yarn-error.log
 screenshots
 .idea
+package-lock.json
```

Comment:
Why have you added and ignored package-lock? If there is one, we'd want to include it.